app/src/main/res/values/donottranslate.xml

```diff
@@ -36,6 +36,7 @@
 
     <string-array name="language_entries">
         <item>@string/system_default</item>
+        <item>British English</item>
```

Comment:
Should we call the other English American English?

Reply:
I'd go with
* English (UK)
* English (US)
so that they show up under `E` where you expect to find English, but this is a matter of opinion, not correctness

Reply:
Thats a good idea, thx!